html-forms/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Links</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="">
+      <section>
+        <div>
+        <label for="email">Email</label>
+        <p><input id="email" type="email" name="email"></p>
+      </div>
+      </section>
+
+      <section>
+        <p>
+        <label for="select"></label>
+          <span>Why is coding cool?</span>
+      </p>
+      <select name="cool" id="coolReason">
```

Comment:
make sure the label on line 18's `for` attribute matches the `id` of this `select` element so that the HTML document knows they are linked.